lib/msal-common/src/utils/Constants.ts

```diff
@@ -204,3 +204,8 @@ export enum GrantType {
     DEVICE_CODE_GRANT = "device_code"
 };
 
+/**
+ * Initialize B2CTrustedHostList
+ */
+export const B2CTrustedHostList = {};
```

Comment:
Remove this since we moved it to AuthorityFactory

Reply:
Done. Thanks, I missed this. :)